doc/source/tutorial/linalg_batch.md

```diff
@@ -0,0 +1,153 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+orphan: true
+---
+
+```{eval-rst}
+.. jupyterlite:: ../../_contents/linalg_batch.ipynb
```

Comment:
I think this should be
```suggestion
.. jupyterlite:: ../_contents/linalg_batch.ipynb
```
because we are under `source/tutorials/` and not under `source/tutorials/stats`